### ISSUE TITLE
[Bug]: Objects().get() should detect error before submission

### ISSUE THREAD

**Issue by azaroth42** [project-lux/luxy#1]
### Priority Level

Medium

### What happened?

```
>>> from luxy import Objects
>>> Objects().get()
Could not find total items in response
<luxy.api.Objects object at 0x1036e27b0>
>>> r.json
{'statusCode': 400, 'errorMessage': 'Invalid search request: search term does not specify a term name in criteria {}'}
```

get() should detect that there's no query and fail before submission

**Comment by wjbmattingly** [comment]
Thanks! Fixed in 0.0.2 and will be merged into main soon.

**Comment by wjbmattingly** [comment]
Closed with #4
